packages/containers/src/ComponentForm/ComponentForm.test.js

```diff
@@ -255,19 +255,22 @@ describe('ComponentForm', () => {
 			const dispatch = jest.fn();
 			const oldUrl = 'http://old';
 			const newUrl = 'http://new';
+			const properties = { properties: { name: 'old' } };
```

Comment:
```suggestion
			const state = { properties: { name: 'old' } };
```

Reply:
state is already take, I rename by componentState